charts/streams-app/templates/jmx-configmap.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if and .Values.prometheus.jmx.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "streams-app.name" . }}-jmx-configmap
+  labels:
+    app: {{ template "streams-app.name" . }}
+    chart: {{ template "streams-app.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  jmx-kafka-streams-app-prometheus.yml: |+
+    jmxUrl: service:jmx:rmi:///jndi/rmi://localhost:{{ .Values.jmx.port }}/jmxrmi
+    lowercaseOutputName: true
+    lowercaseOutputLabelNames: true
+    ssl: false
+    rules:
+    - pattern: ".*"
```

Comment:
We should make this configurable. In case someone wants to limit the metric space.